resources/linux/debian/postinst.template

```diff
@@ -116,6 +132,10 @@ NdCFTW7wY0Fb1fWJ+/KTsC4=
 =J6gs
 -----END PGP PUBLIC KEY BLOCK-----
 " | gpg --dearmor > $CODE_TRUSTED_PART
+			OLD_KEY_PATH="/etc/apt/trusted.gpg.d/microsoft.gpg"
```

Comment:
Could you place this constant below CODE_TRUSTED_PART and rename it to CODE_TRUSTED_PART_OLD?